pyproject.toml

```diff
@@ -38,6 +30,17 @@ cu12 = [
     "nvidia-cuda-nvrtc-cu12",
     "nvidia-nvjitlink-cu12"
 ]
+# TODO: Use cuda-toolkit package dependencies - e.g. cuda-toolkit[curand,nvvm,nvrtc]=13.*
+cu13 = [
+    "cuda-bindings==13.*",
+    "cuda-core==0.3.2,<0.4.0dev0",
```

Comment:
In the near future we should aim to relax this pinning, I think it's too tight. The cuda-core features used by numba-cuda has been stabilized since 0.3.0.